integration_tests/support/commands.js

```diff
@@ -15,6 +21,24 @@ Cypress.Commands.add('withinFieldsetThatContains', (text, action) => {
   cy.contains(text).parent('fieldset').within(action)
 })
 
+Cypress.Commands.add('stubViewReferralDetails', referralToView => {
```

Comment:
This is a great idea....

What do you think of making it into a discrete action of actually viewing the referral details page? 

This simply requires renaming it to viewReferralDetailsPage
and adding the command 
cy.visit(`/service-provider/referrals/${referralToSelect.id}/details`)
at the bottom